.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v3
         with:
-          dotnet-version: 3.1.x
```

Comment:
Remove
```yml
with:
  dotnet-version: 6.0.x
```

From all the workflow files changed. The `actions/setup-dotnet` action will read `global.json` to find the SDK version.

Reply:
done.